### DISCUSSION TITLE
依赖传递问题

### DISCUSSION THREAD

**Discussion by uummxx** [lisonge/vite-plugin-monkey#143] [Q&A]
我现在想排除依赖 但是不使用参数传递模式 我不能直接在 build 里面排除吗？
![Clip_2024-03-28_23-49-47](https://github.com/lisonge/vite-plugin-monkey/assets/97946163/24f2b067-3fe3-4cb8-8f68-c6b4bf1094b7)

原来你的打包是通过 function f(vue) 传递的嘛 我现在不想这样子用 有没有其他什么办法？
![Clip_2024-03-28_23-47-36](https://github.com/lisonge/vite-plugin-monkey/assets/97946163/33487cec-6034-4660-a88c-98140dc32a97)
这是你原来的
![Clip_2024-03-28_23-49-06](https://github.com/lisonge/vite-plugin-monkey/assets/97946163/b292ce94-585a-425e-a295-38cec6d5a863)

**Comment by lisonge** [comment]
1.不想这样用的原因是什么

2.你要实现什么效果？
